blend-library/src/main/java/com/wealthfront/blend/animator/SinglePropertyAnimation.kt

```diff
@@ -29,17 +30,26 @@ class SinglePropertyAnimation<Subject>(
    * The animator running this animation
    */
   var animator: BlendableAnimator? = null
-    private set
+    @VisibleForTesting internal set
+  /**
+   * Whether this animation has started running. Note that it can be committed (i.e. queued) and not started.
```

Comment:
why don't we just use queued everywhere coz that relays to me the information better than committed tbh.

Reply:
Good point, I changed the language everywhere to "queued." And now queue doesn't look like a real word haha.